Tasks/SaveCacheV0/task.json

```diff
@@ -0,0 +1,51 @@
+{
+    "id": "F2333FED-1017-4A2F-AAC6-02A185279E1E",
+    "name": "SaveCache",
+    "friendlyName": "Save Cache",
+    "description": "Publish a local cache for the current pipeline.",
+    "helpUrl": "",
+    "helpMarkDown": "Publish a local cache for the current pipeline.",
+    "category": "Utility",
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 0,
+        "Minor": 1,
+        "Patch": 0
+    },
+    "groups": [],
+    "demands": [],
+    "preview": true,
+    "minimumAgentVersion": "2.140.1",
+    "inputs": [
+        {
+            "name": "key",
+            "type": "multiLine",
+            "label": "The name of the key.",
+            "defaultValue": "key",
+            "required": true,
+            "helpMarkDown": "The name of the key."
+        },
+        {
+            "name": "path",
+            "type": "filePath",
+            "label": "Path to publish",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "The folder or file path to publish. This can be a fully-qualified path or a path relative to the root of the repository. Wildcards are not supported. [Variables](https://go.microsoft.com/fwlink/?LinkID=550988) are supported."
+        },
+        {
+            "name": "salt",
+            "type": "string",
+            "label": "A salt to use for generating a fingerprint.",
```

Comment:
>A salt [](start = 22, length = 6)

looks odd for a label, ask PM for exact string

Reply:
@arunkm , this is a temp place for salt - we will eventually remove it from the task. @mitchdenny - Any idea for the salt label. Thanks